### ISSUE TITLE
Browsing Website as Guest

### ISSUE THREAD

**Issue by pijulius** [opensource-socialnetwork/opensource-socialnetwork#1980]
Hi Lianglee,

First of all thanks for working on this open source social network and providing it free of charge!

I have set up the dev version for testing but one thing I can't figure out is that is it possible to browse the website (especially the home/ wall part) as guest? I mean could it be made to have a public home feed that guests can view and browse but only be able to  comment/like and so on once they logged in?

I read something about a "Private Network" component that if disabled would allow this to happen but I don't have that component and not sure if installing and disabling would do anything :)

Bottom line: could you please let me know if it's possible to make home/ accessible by guests and if so how?

Thanks in advance!
Julius
